docs/user/visualize.asciidoc

```diff
@@ -15,31 +15,46 @@ _Visualize_ enables you to create visualizations of the data from your Elasticse
 . Click *Create new visualization*.
 . Choose the visualization type:
 +
-* *Basic charts*
-<<lens,Lens>>:: Quickly build several types of basic visualizations by simply dragging and dropping the data fields you want to display.
+* <<lens,Lens>>
```

Comment:
Need some punctuation after the word Lens.

Although I understand the linking in this section, it's a little confusing because it isn't consistent in how its used.